testutils/src/main/scala/za/co/absa/ComparisonJob.scala

```diff
@@ -0,0 +1,65 @@
+package za.co.absa
+
+import org.apache.spark.sql._
+import org.apache.spark.SparkContext
+import org.apache.spark.sql.types.{StructField, StructType}
+import za.co.absa.enceladus.standardization.CmdConfig
+
+object ComparisonJob {
+  def getReader(cmd: CmdConfig)(implicit sparkSession: SparkSession): DataFrameReader = {
+    val dfReader = sparkSession.read.format(cmd.rawFormat)
+    // applying format specific options
+    val  dfReader4= {
+      val dfReader1 = if (cmd.rowTag.isDefined) dfReader.option("rowTag", cmd.rowTag.get) else dfReader
+      val dfReader2 = if (cmd.csvDelimiter.isDefined) dfReader1.option("delimiter", cmd.csvDelimiter.get) else dfReader1
+      val dfReader3 = if (cmd.csvHeader.isDefined) dfReader2.option("header", cmd.csvHeader.get) else dfReader2
+      dfReader3
+    }
+    if (cmd.rawFormat.equalsIgnoreCase("fixed-width")) {
+      val dfReader5 = if (cmd.fixedWidthTrimValues.get) dfReader4.option("trimValues", "true") else dfReader4
+      dfReader5
+    } else dfReader4
+  }
+
+  def removeMetadataFromSchema(schema: StructType): StructType = {
+    StructType(schema.map{ f =>
+      StructField(f.name, f.dataType, f.nullable)
+    })
+  }
+
+  def main(args: Array[String]): Unit = {
+    val cmd = CmdConfig.getCmdLineArguments(args)
+    val enableWholeStage = false
+
+    implicit val sparkSession: SparkSession = SparkSession.builder()
+      .appName("ABSA QA Test Utils")
+      .config("spark.sql.codegen.wholeStage", enableWholeStage) //disable whole stage code gen - the plan is too long
+      .getOrCreate()
+
+    implicit val sc: SparkContext = sparkSession.sparkContext
+
+    import sparkSession.implicits._
+
+    val expected = getReader(cmd).load(cmd.refPath)
+    val actual = getReader(cmd).load(cmd.stdPath)
```

Comment:
```suggestion
    val actualDf = getReader(cmd).load(cmd.stdPath)
```
we have expectedSchema and actualScehma, makes sense to elaborate on the other expected and actual entities